examples/gnn_link.py

```diff
@@ -27,7 +27,7 @@
 parser.add_argument("--dataset", type=str, default="rel-hm")
 parser.add_argument("--task", type=str, default="rel-hm-rec")
 parser.add_argument("--lr", type=float, default=0.001)
-parser.add_argument("--epochs", type=int, default=20)
+parser.add_argument("--epochs", type=int, default=5)
```

Comment:
Why do we shorten it? Is it enough to converge?

Reply:
Reverted